lithium/lithium.py

```diff
@@ -141,7 +143,8 @@ def __init__(self):
         self.cutBefore = self.DEFAULT_CUT_BEFORE
 
     def readTestcaseLine(self, line):
-        cutter = b"[" + self.cutBefore + b"]?[^" + self.cutBefore + self.cutAfter + b"]*(?:[" + self.cutAfter + b"]|$|(?=[" + self.cutBefore + b"]))"
+        cutter = b"[" + self.cutBefore + b"]?[^" + self.cutBefore + self.cutAfter + b"]*(?:[" + self.cutAfter + \
+            b"]|$|(?=[" + self.cutBefore + b"]))"
```

Comment:
Personally I like this style better (brackets and no `\`), but I'll leave it up to you:

    cutter = (b"[" + self.cutBefore + b"]?" +
              b"[^" + self.cutBefore + self.cutAfter + b"]*" +
              b"(?:[" + self.cutAfter + b"]|$|(?=[" + self.cutBefore + b"]))")